### ISSUE TITLE
Permission denied when trying to compile

### ISSUE THREAD

**Issue by SmallRoomLabs** [chunkypixel/atari-dev-studio#1]
Using VC version 1.33.1 on Mojave 10.14.5 I get this when I try to compile some code (by pressing the "Compile source code" button at the bottom status line)

`Starting build of fluuw.a...
/bin/sh: /Users/mats/.vscode/extensions/chunkypixel.atari-dev-studio-0.1.0/out/bin/compilers/dasm/dasm.Darwin.x86: Permission denied`

The issue persists even after a restart of code (tried that since older versions of code required a restart after installation of extentions)

**Comment by SmallRoomLabs** [comment]
Easy fix for the time being...

`chmod a+x ~/.vscode/extensions/chunkypixel.atari-dev-studio-0.1.0/out/bin/compilers/dasm/dasm.Darwin.x86`

**Comment by chunkypixel** [comment]
Hey thanks for the report! I have added some chmod permission checks which I thought should have covered this off (does on Linux which I have checked myself). I'll check that out very soon. Thanks again.

**Comment by chunkypixel** [comment]
Reviewed in build v.0.1.1
I've reviewed the code and it should be working (it may not follow the same permissions process as Linux??? -  this code has previously worked in my other batari Basic extension for another mac user).  I've updated the notification to also show in the Compile log instead of just the Console log (which is available via the Developer Tools). I may need to access my brothers mac to fix this one...

**Comment by SmallRoomLabs** [comment]
I can remove the extension and then re-install and see what happens. Are there any log that shows the commands/steps being executed during the installation?

**Comment by chunkypixel** [comment]
None that I am aware of unfortunately or can find. Even using the extension I couldn't find any logs so I output stuff to the console which you can see via the Help -> Developer Tools (select the console tab). Ive added the actual error now for permission to the compiler output.

**Comment by SmallRoomLabs** [comment]
I'll try it when I get back home later tonight...

**Comment by SmallRoomLabs** [comment]
You've got an extra '/dasm.Darwin.x86' on the chmod line..

```
failed to set chmod permissions: ENOTDIR: not a directory, chmod '/Users/mats/.vscode/extensions/chunkypixel.atari-dev-studio-0.1.1/out/bin/compilers/dasm/dasm.Darwin.x86/dasm.Darwin.x86'
Starting build of fluuw.a...
/bin/sh: /Users/mats/.vscode/extensions/chunkypixel.atari-dev-studio-0.1.1/out/bin/compilers/dasm/dasm.Darwin.x86: Permission denied

```

**Comment by chunkypixel** [comment]
Fixed in build v0.1.2
Sorry I had refactored this code which duplicated the filename.
